src/components/LayerTree/LayerTree.js

```diff
@@ -240,26 +221,29 @@ class LayerTree extends Component {
   // Render a button which expands/collapse the layer if there is children
   // or simulate a click on the input otherwise.
   renderToggleButton(layer) {
-    const { t, classNameToggle, buttonTitles } = this.props;
+    const { t, titles } = this.props;
     const { expandedLayerNames } = this.state;
-    const tabIndex = 0;
-
+    const onInputClick = () => {
+      this.onInputClick(layer, layer.getChildren().length);
+    };
+    const title = `${t(layer.getName())} ${
+      !expandedLayerNames.includes(layer)
+        ? titles.subLayerShow
+        : titles.subLayerHide
+    }`;
     return (
-      <Button
-        tabIndex={tabIndex}
-        title={`${t(layer.getName())} ${
-          !expandedLayerNames.includes(layer)
-            ? buttonTitles.subLayerShow
-            : buttonTitles.subLayerHide
-        }`}
-        className={classNameToggle}
-        onClick={() => {
-          this.onInputClick(layer, layer.getChildren().length);
-        }}
+      <div
+        role="button"
+        tabIndex={0}
+        className="rs-layer-tree-toggle"
+        title={title}
```

Comment:
Could we add here 'aria-expanded' for accessibility:
https://www.accessibility-developer-guide.com/examples/sensible-aria-usage/expanded/

Reply:
```suggestion
        title={title}
        aria-expanded={expandedLayerNames.includes(layer)}
```